desktop/test/e2e/warnings/warnings.spec.ts

```diff
@@ -486,6 +486,28 @@ test.describe('warnings --', () => {
     });
 
 
+    test('disable multiple switch if single resource is affected by unconfigured category warning', async () => {
+
+        await waitForNotExist(await NavbarPage.getWarnings());
+        await createUnconfiguredCategoryWarnings(['1'], 'CustomCategory');
+
+        expect(await NavbarPage.getNumberOfWarnings()).toBe('1');
+
+        await NavbarPage.clickWarningsButton();
+        await WarningsModalPage.clickSelectNewCategoryButton(0);
+        await CategoryPickerPage.clickSelectCategory('Layer', 'Feature');
+        await expect(SelectModalPage.getMultipleSwitch()).toBeDisabled();
```

Comment:
This should be written as:
expect(await SelectModalPage.getMultipleSwitch()).toBeDisabled();